src/Sulu/Bundle/ContentBundle/Behat/ContentContext.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle\Behat;
+
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\PyStringNode;
+use Behat\Behat\Hook\Scope\AfterScenarioScope;
+use Sulu\Component\Content\StructureInterface;
+use Sulu\Component\Content\Mapper\ContentMapperRequest;
+use Behat\Gherkin\Node\TableNode;
+use Sulu\Bundle\ContentBundle\Behat\BaseStructureContext;
+
+/**
+ * Behat context class for the ContentBundle
+ */
+class ContentContext extends BaseStructureContext implements SnippetAcceptingContext
+{
+    /**
+     * @Given there exists a page template :arg1 with the following property configuration
+     */
+    public function thereExistsAPageTemplateWithTheFollowingPropertyConfiguration($arg1, PyStringNode $string)
+    {
+        $template = <<<EOT
+<?xml version="1.0" ?>
+
+<template xmlns="http://schemas.sulu.io/template/template"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://schemas.sulu.io/template/template http://schemas.sulu.io/template/template-1.0.xsd"
+          >
+
+    <key>%s</key>
+
+    <view>ClientWebsiteBundle:templates:%s</view>
+    <controller>SuluWebsiteBundle:Default:index</controller>
+    <cacheLifetime>2400</cacheLifetime>
+
+    <meta>
+        <title lang="de">%s</title>
+        <title lang="en">%s</title>
+    </meta>
+
+    <properties>
+        <property name="title" type="text_line" mandatory="true">
+            <meta>
+                <title lang="de">Tier</title>
+                <title lang="en">Animals</title>
+            </meta>
+
+            <tag name="sulu.rlp.part"/>
+        </property>
+
+        <property name="url" type="resource_locator" mandatory="true">
+            <meta>
+                <title lang="de">Adresse</title>
+                <title lang="en">Resourcelocator</title>
+            </meta>
+
+            <tag name="sulu.rlp"/>
+        </property>
+
+%s
+    </properties>
+</template>
+EOT
+        ;
```

Comment:
The php documentation says differently: http://php.net/manual/de/language.types.string.php#language.types.string.syntax.heredoc